testutils/src/main/java/com/amplifyframework/testutils/Assets.java

```diff
@@ -79,5 +81,30 @@ public static Bitmap readAsBitmap(final String name) throws RuntimeException {
             throw new RuntimeException("Failed to load asset " + name, ioException);
         }
     }
-}
 
+    /**
+     * Lists the files in the assets directory.
+     * @param path Directory path under assets/.
+     * @return A list of files found
+     */
+    @NonNull
+    public static List<String> list(String path) {
```

Comment:
This utility is used by the `SchemaLoader`. The schema loader reads the schema JSON from disk with this `list(...)` utility. It will then build `ModelSchema` instances, from each.